packages/vue-docgen-api/tests/components/vuetify/vuetify.test.ts

```diff
@@ -0,0 +1,23 @@
+import * as path from 'path'
+import { ComponentDoc } from '../../../src/Documentation'
+import { parse } from '../../../src/main'
+
+
+const field = path.join(__dirname, './VTextField.js')
+let docField: ComponentDoc
+
+describe('tests button with pure javascript', () => {
+	beforeAll(async done => {
+		docField = await parse(field)
+		done()
+	})
+
+	it('should extract the export name', () => {
+		expect(docField.exportName).toBe('default')
+	})
+
+	it('should has props', () => {
+		// expect(docField.props).not.toBeUndefined()
```

Comment:
There is actually a big misunderstanding in your tests.
Can you look at them please?

```suggestion
		expect(docField.props).not.toBeUndefined()
```